contact.html

```diff
@@ -87,17 +87,15 @@
             <li class="nav-item">
               <a class="nav-link" href="profile.html">Profile</a>
             </li>
+            <li class="nav-item">
+              <a class="nav-link" href="contact.html">Contact</a>
+            </li>
           </ul>
-          <form class="form-inline my-2 my-lg-0 pull-right">
-            <input
-              class="form-inline mr-sm-2"
-              type="search"
-              placeholder="Search"
-              aria-label="Search"
-            />
-            <button class="btn btn-outline-primary my-2 my-sm-0" type="submit">
-              Search
-            </button>
+          <form >
+            <div>
+              <input class="form-inline my-2 mr-sm-2" type="search" placeholder=" Search" aria-label="Search">&nbsp;&nbsp;
```

Comment:
- In placeholder the space will not act as a space you can use &nbsp if you want to add space there

Reply:
- I don't got why you added 2 spaces after input